R/interface.R

```diff
@@ -289,9 +289,16 @@ sourceExamples <- function(file){
 rcppoctave <- function(..., plugin = 'RcppOctave'){
 	if( !requireNamespace('inline') )
 		stop("Package 'inline' is required to define RcppOctave functions.")
-	#qlibrary(inline)
-	#rcpp(..., plugin = plugin)
-	cxxfunction(..., plugin = plugin)
+	#qlibrary('inline')
+  # Fix issue when runnin in unit tests
```

Comment:
These lines are indented by two *spaces* rather than two *tabs*, like the rest of this file. Most of the code quite consistently uses *tabs*.